src/components/Popups/PopupDrawingToolbar.scss

```diff
@@ -1,5 +1,40 @@
 @import '~box-ui-elements/es/styles/variables';
 
+@mixin PopupDrawingToolbarButton {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 32px;
+    color: $white;
+    background: transparent;
+    border: none;
+    border-radius: $bdl-border-radius-size;
+    outline: none;
+    cursor: pointer;
+    opacity: .7;
+    transition: opacity 200ms ease;
+
+    &:focus,
+    &:hover {
+        opacity: 1;
+    }
+
+    &[disabled] {
+        cursor: default;
+        opacity: .2;
+    }
+}
+
+@mixin PopupDrawingToolbarIconButton {
+    @include PopupDrawingToolbarButton;
+
+    width: 32px;
```

Comment:
It seems a little odd for the height and width to share the same value but be declared in different mixins. We could make both height and width an argument of `PopupDrawingToolbarButton` and include a default value of 32px each. It would likely eliminate the need for this separate icon-specific mixin.